ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -30,5 +30,281 @@
 #include <catch2/catch.hpp>
 
 #include "azure_connection.h"
+#include <fstream>
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("Testing Azure Connection Class", "azure-connection")
+{
+
+    Azure::Storage::Blobs::BlobContainerClient azure_client =
```

Comment:
we can use auto here.

Reply:
resolved here https://github.com/wiredtiger/wiredtiger/pull/8747/commits/b8979db4be25c1b70cd0fc6ceba12047b1b27c25